### ISSUE TITLE
libmxnet.so: undefined symbol: sgeqrf_

### ISSUE THREAD

**Issue by liuzhi136** [apache/incubator-mxnet#20081]
Platform Infos:
Ubuntu20.04+V100+cuda11
I had successfully installed the cuda environment. Then, I use the pip to install mxnet-cu110(1.8.0).
The error "OSError: /home/lz/anaconda3/envs/ML_gpu/lib/python3.6/site-packages/mxnet/libmxnet.so: undefined symbol: sgeqrf_" is occured when I import mxnet.

I saw some ways to fix this issues, such as 'make clean_all'. However, such way is used to fix this issue when we build mxnet from source.
How can I fix this issue for 'pip install'

**Comment by leezu** [comment]
Did you ever try compile mxnet from source before? If so, you probably didn't uninstall it completely

**Comment by liuzhi136** [comment]
@leezu No, cuz I never clone the mxnet source code by git completely. It's to slow and always occurs the time out error.

**Comment by szha** [comment]
@liuzhi136 `sgeqrf_` is a lapack symbol. For linux, we've been statically linking it from openblas (@leezu keep me honest here). So I'd lean towards a "left-over installation hypothesis" as Leo mentioned too. Did you try using it without conda environment?
